### ISSUE TITLE
Enable Newsletter Sign Up Box on MozFest Site

### ISSUE THREAD

**Issue by cdanfon** [mozilla/foundation.mozilla.org#9436]
Enable **Newsletter Sign Up Box** on MozFest Site by re-using #8787 however, this needs to be treated as a streamfield instead of as part of the page as per #8787 

The signup box should link to the MozFest Signup

**Comment by cdanfon** [comment]
Creating this ticket from the back of this request: https://airtable.com/appXJat3iETgLcZOH/tblmw0OQI3m4sRu1x/viw6ThkBXLHKqmQN1/recPYqor9afeZ6FMq?blocks=bipUae6IzTju49vpU

Awaiting further details before we can groom.

**Comment by bheasman** [comment]
Adding more context here as to what's required (essentially creating a new streamfield component for adding a newsletter sign up form on Mozfest pages):

- Design to reuse the hardcoded component at the bottom of blog pages - [blog newsletter component](https://github.com/mozilla/foundation.mozilla.org/issues/8787)
- Add it as a new streamfield component to mozfest_homepage and mozfest_primary_page so that it can be added anywhere within the page body content
- Within the component, editors can just pick from the list of signup_snippets - which will include the existing Mozfest one that is used in the signup field.

**Comment by tomusher** [comment]
This is done and ready for review in #9592

**Comment by cdanfon** [comment]
@marcwalsh ready for you to test!

**Comment by kristinashu** [comment]
It might just be that I'm tired but I don't see the block on staging? What is it called?

<img width="1007" alt="image" src="https://user-images.githubusercontent.com/1068783/202576482-0215862e-4038-41ef-9121-ff2ae6f26fd3.png">

**Comment by danielfmiranda** [comment]
Hi @cdanfon! While the PR for this ticket has been approved, it actually has not been merged into the codebase yet as there are a couple merge conflicts waiting to be fixed. Once those are done we can then merge and push to staging!

**Comment by tomusher** [comment]
I've resolved those conflicts so this should be ready to merge now @danielfmiranda

**Comment by cdanfon** [comment]
Hey @tbrlpld & @danielfmiranda can you check if this has been deployed to prod? From the ticket status (closed) I'd assume so, but @marcwalsh cannot see it on the prod environement. Thanks!

**Comment by tbrlpld** [comment]
@cdanfon I think this was accidentally closed through the PR on Friday. Should be on staging, but no production deployments since.

**Comment by cdanfon** [comment]
@tbrlpld Hey Tibor, just to check - if closing a PR does that mean the ticket automatically gets closed?

**Comment by tbrlpld** [comment]
@cdanfon It does when there is a keyword used to signify that it does close the issue, like `closes #9592`. Other keywords are `resolves`, `fixes` etc. 

I had updated the PR template to **not** include these keywords anymore to make sure PRs fit into out overall workflow.

See also : https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/using-keywords-in-issues-and-pull-requests

**Comment by tbrlpld** [comment]
Wow.. did not notice I accidentally closed it yesterday too...

**Comment by fessehaye** [comment]
On prod
